build.sh

```diff
@@ -17,7 +17,7 @@ ORGANISATION_DOMAIN=${ORGANISATION_DOMAIN:-example.com}
 PKI_NAME="trust.$ORGANISATION_DOMAIN"
 ROOT_PKI_NAME=${ROOT_PKI_NAME:-$PKI_NAME}
 INTERMEDIATE_PKI_NAME=${INTERMEDIATE_PKI_NAME:-$PKI_NAME}
-OUTPUT_DIR=${OUTPUT_DIR:-.}
+OUTPUT_DIR=${OUTPUT_DIR:-$PWD}
```

Comment:
I wonder if it is practical to use `readlink -f $OUTPUT_DIR` to turn relative links into full paths so the caller does not have to be concerned about such trifles.

Reply:
Done.